src/streams/stransport.c

```diff
@@ -84,6 +84,7 @@ static int stransport_connect(git_stream *stream)
 
 	if (sec_res == kSecTrustResultDeny || sec_res == kSecTrustResultRecoverableTrustFailure ||
 	    sec_res == kSecTrustResultFatalTrustFailure)
```

Comment:
I think you may want braces for this if statement now that it's two expressions?

Reply:
Yeah right 😆 . I miss my `goto fail`s so much.